.pipelines/ci/templates/build-powertoys-steps.yml

```diff
@@ -12,6 +12,13 @@ steps:
   submodules: true
   clean: true
 
+- task: PowerShell@2
+  displayName: Verify Notice.md and Nuget packages match
+  inputs:
+    filePath: '$(build.sourcesdirectory)\.pipelines\verifyNoticeMdAgainstNugetPackages.ps1'
```

Comment:
running this from solution dir works, just pass current dir as an arg :) `.\.pipelines\verifyNoticeMdAgainstNugetPackages.ps1 .`

Reply:
This is going to be removed. It was here to test why it was failing asap